site/en/blog/floc/index.md

```diff
@@ -22,6 +22,17 @@ history. The user's browser is associated with one interest cohort at a time and
 cohort periodically (currently once every seven days during this initial origin trial) on 
 the user's device, without sharing individual browsing data with the browser vendor or anyone else. 
 
+{% Aside %}
+A page visit will only be included in the browser's FLoC calculation if `document.interestCohort()` 
+is used on the page. In other words, your cohort is based only on the pages you've visited that use 
+FLoC. 
```

Comment:
Same comment as other article, I think this implies more certainty about the outcome of the origin trial than we have. Should probably just state the inclusion criteria as it applies to the origin trial.

Reply:
Updated as per other article.